nb/projects/pentacular/tiles/hex.md

```diff
@@ -0,0 +1,380 @@
+```JavaScript
+const hex = Hexagon(32);
+```
+
+```JavaScript
+const Tab = (thickness, tabLength) =>
```

Comment:
I feel like tabs are a common enough thing that I do that it might be worth having a stand alone tabs library that we could import and use for things like this

Reply:
Although it's not that hard to do them as is

Reply:
Yes, I think so.

I think we're a bit weak on geometric introspection at the moment.

I'll try to get better support for finding where rays intersect volumes, etc, for more flexible placement.